src/workflow/ArcanistPatchWorkflow.php

```diff
@@ -1123,4 +1124,22 @@ private function buildDependencyGraph(ArcanistBundle $bundle) {
     return $graph;
   }
 
+  private function pullBaseTagFromStagingArea($id){
+    list($success, $message, $staging, $staging_uri) = $this->validateStagingSetup();
+    if (!$success) {
+      return $message;
+    }
+    $prefix = idx($staging, 'prefix', 'phabricator');
+    $base_tag = "{$prefix}/base/{$id}";
+    echo pht('Fetching base tag from staging remote')."\n";
+    $err = phutil_passthru(
+          'git fetch --tag -n %s %s',
```

Comment:
Does this work even if the remote is not added? That will be the workflow on local developer laptops